### PR TITLE
Feature cleanup template

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**

### CODE REVIEW

File: src/app/index.html [hunk 1]
```diff
@@ -1 +1,51 @@
...
+        <p>
+          For tutorials, information, and recomendations using Stache to create your documentation, please refer to the <a href="https://github.com/blackbaud/stache2-docs">Stache Docs</a>.
+        </p>
+    </stache-column>
+    <stache-column screenSmall="12" screenMedium="4" screenLarge="4">
+      <stache-page-anchor>
+        Service Catalog
+      </stache-page-anchor>
+        <p>
+          Once your documentation site is deployed and ready for consumption, don't forget to add a link to your docs in the <a href="https://service-catalog.docs.blackbaud.com/#/">Service Catalog</a>.
```
Comment (blackbaud-brandonhare) [inline src/app/index.html:43]: Yus

File: src/app/learn/getting-started/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<app-nav></app-nav>
+<stache
+  pageTitle="Getting Started"
+  showTableOfContents="true">
+
+  <stache-page-summary>
+    <p>
+      Stache provides an excellent means of spinning up documentation for your products and services. A learn section is a great area to teach others how to use your stuff.
```
Comment (blackbaud-brandonhare) [inline src/app/learn/getting-started/index.html:8]: Same text as the learn page?

File: src/app/learn/resources/index.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<app-nav></app-nav>
+<stache pageTitle="Resources">
+
+  <stache-page-summary>
+    <p>
+      Resources is an excellent place to provide links to further documetnation that help support your product or service.
```
Comment (blackbaud-brandonhare) [inline src/app/learn/resources/index.html]: Maybe: "Use resources to provide additional links, supplemental information, and any other pertinent details that support your service."

**Review by blackbaud-brandonhare — APPROVED** [review]
Format looks good. I would just have a writer review for content. Noice!

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**